### ISSUE TITLE
Symfony 4.1 twig error : Variable "help" does not exist

### ISSUE THREAD

**Issue by db306** [symfony/symfony#27489]
**Symfony version(s) affected**: 4.1

**Description**  
When moving from 4.0.11 to 4.1 Twig returns an error `Variable "help" does not exist` This error is triggered on the form templates. This error appears when using bootstrap, foundation 5 and the default layout.

**How to reproduce**  
Create a simple form using the FormBuilder:

```
    public function buildForm(FormBuilderInterface $builder, array $options)
    {

        $builder
            ->add('email', EmailType::class, array (
                'label' => 'user.form.email'
            ))
            ->add('username', TextType::class, array (
                'label' => 'user.form.username'
            ))
            ->add('plainPassword', RepeatedType::class, array(
                'type' => PasswordType::class,
                'options' => array(
                    'translation_domain' => 'User',
                    'attr' => array(
                        'class' => 'edit-optional-password'
                    )
                ),
                'first_options' => array('label' => 'user.form.password'),
                'second_options' => array('label' => 'user.form.password_confirmation'),
                'invalid_message' => 'user.password.mismatch',
                'attr' => array(
                    'data-del' => 'del'
                )
            ))
            ->add('register', SubmitType::class, array (
                'label' => 'user.form.register.button'
            ));
    }
```

**Possible Solution**  
For example in the file `vendor/symfony/symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig`

Line 262 and Line 300
From : `{%- if help is not empty -%}`
To: `{%- if help is defined and help is not empty -%}`

**Additional context**

**Comment by db306** [comment]
I sent a PR #27490

**Comment by xabbuh** [comment]
I think we should first investigate why `help` isn't set in your case and see if we can fix that instead. Could you create a small example application that allows to reproduce your issue?

**Comment by db306** [comment]
@xabbuh I'll create an application example that reproduces this bug

**Comment by yceruto** [comment]
https://github.com/symfony/symfony/blob/9660103ff1141e05591f5064382c4efece491736/src/Symfony/Component/Form/Extension/Core/Type/FormType.php#L92

`help` var should always be defined. Is there a possibility that you've installed `symfony/twig-bridge` (4.1) and an earlier version of `symfony/form`?

**Comment by db306** [comment]
@yceruto updating `symfony/form` fixed the issue. I thought that updating to symfony 4.1 would update its dependencies or fail because it requires `symfony/form` to have a higher version as it does for many other packages.
Thanks for the feedback

**Comment by Nek-** [comment]
@db306 please close this issue then. :)

**Comment by xabbuh** [comment]
I wonder how this could actually happen as #26332 which is part of the 4.1 release of the TwigBridge should make sure that the Form component is installed in a compatible version.

**Comment by loicpennamen** [comment]
Solution in my case: I had the same issue, somehow I figured my form type's name was the problem. On Symfony 6.3 with all updated dependencies, I created an `App\Form\Type\PercentType` class (basically a glorified `IntegerType`).

The bug happens.

When I change the name of my class to `App\Form\Type\PctType` the error goes away. It is quite a funny bug, unfortunately no time right now to dig deeper into a fix proposal for Symfony.
